kitsune/bundles.py

```diff
@@ -243,15 +243,6 @@
         "source_filenames": ("d3/d3.js", "kpi/js/kpi.browserify.js",),
         "output_filename": "build/kpi.dashboard-min.js",
     },
-    "experiment_fxa_cta_topbar": {
-        "source_filenames": (
-            "sumo/js/libs/mozilla-dnt-helper.js",
```

Comment:
This library isn't used anywhere else and can be deleted, we use "kitsune/sumo/static/sumo/js/dnt-helper.js" for things like GA.